.github/workflows/vale.yml

```diff
@@ -0,0 +1,25 @@
+name: Check Prose with Vale
+
+on:
+  pull_request:
+    paths:
+      - '**/*.md'
+  push:
+    paths:
+      - '**/*.md'  # You can specify other file extensions or directories if needed
+
+jobs:
+  vale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up Vale
+        run: |
+          curl -sSL https://github.com/errata-ai/vale/releases/download/v2.12.0/vale_2.12.0_Linux_64-bit.tar.gz | tar -xz -C /tmp
```

Comment:
It seems the latest version is [v3.9.1](https://github.com/errata-ai/vale/releases/tag/v3.9.1), is there a specific reason you're using [v2.12.0](https://github.com/errata-ai/vale/releases/tag/v2.12.0)?